bin/clean-ninja.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const path = require('path')
+const fs = require('fs-extra')
+
+// Clean ninja related files
+const cleanFiles = async () => {
+  // clean images, icons and fonts
+  await fs.emptyDir(path.resolve(__dirname, '../src/images'))
+  await fs.emptyDir(path.resolve(__dirname, '../src/images/svgs'))
+  await fs.emptyDir(path.resolve(__dirname, '../src/icons'))
+  await fs.emptyDir(path.resolve(__dirname, '../src/fonts'))
+
+  // clean static folder
+  await fs.remove(path.resolve(__dirname, '../src/static/favicon-template.png'))
+  await fs.remove(path.resolve(__dirname, '../src/static/favicon.ico'))
+  await fs.remove(path.resolve(__dirname, '../src/static/pin-icon.svg'))
+  await fs.remove(path.resolve(__dirname, '../src/static/social.jpg'))
+  await fs.remove(path.resolve(__dirname, '../src/static/touch-icon.png'))
+  await fs.remove(path.resolve(__dirname, '../src/static/touch-startup-image.png'))
```

Comment:
Vezi ca nu-i bine ce faci. Ar trebui sa inlocuiesti astea cu placeholder files (mai ales in static unde oamenii tb sa genereze favicons si socials si trebuie sa aiba un starting point. 

La fel, cred ca in majoritatea folderelor alea trebuie sa fie ceva sa nu dea eroare la build.